diff --git a/makefile b/makefile
index 145a32f3b..310d32799 100644
--- a/makefile
+++ b/makefile
@@ -12,8 +12,8 @@ PFLOTRANCHEM_LIB="${PFLOTRAN_DIR_LOC}/src/pflotran/libpflotranchem.a"
 # build pflotran and libpflotran.a
 all:
 	@(cd ${PFLOTRAN_DIR_LOC}/src/pflotran && \
-           ${MAKE} pflotran libpflotran.a libpflotranchem.a PETSC_DIR=${PETSC_DIR_LOC} PETSC_ARCH=${PETSC_ARCH_LOC}) 2>&1 | tee make.log
-	@if [ -e "${PFLOTRAN_EXE}" -a -e "${PFLOTRAN_LIB}" -a -e "${PFLOTRANCHEM_LIB}" ] ; then \
+           ${MAKE} pflotran PETSC_DIR=${PETSC_DIR_LOC} PETSC_ARCH=${PETSC_ARCH_LOC}) 2>&1 | tee make.log
+	@if [ -e "${PFLOTRAN_EXE}" ] ; then \
           echo "pflotran, libpflotran.a, and libpflotranchem.a successfully built." ;\
         else \
           echo "failed building pflotran, libpflotran.a, and/or libpflotranchem.a"; exit 1 ;\
